elastic/integrations
# Tenable_sc: Update API lastseen parameter format

## Proposed commit message
The API documentation for `vulnerability` mentions the `lastSeen` parameter format should be `#:#` where `#` is in number of days.
Although the existing format `@-@` where `@` is epoch seconds is still working, this discrepancy is addressed to avoid any future issues.

Other changes:
- Updated request's `header.User-Agent` versions in input files.

Ref: https://github.com/elastic/integrations/issues/10419
API doc: https://docs.tenable.com/security-center/best-practices/api/Content/RetrieveVulnerabilityDataForSpecificTimeRange.htm



## Checklist


## Author's Checklist


- [x] Verify existing and new formats result in same number of documents.

## How to test this PR locally
1. Verified the number of documents matched between current version and new change version with 3 different values:
  a. Initial Interval as `24h`
  b. Initial Interval as `48h`
  c. Initial Interval as `4h`
1. Run pipeline tests for package. Should pass successfully.
`eval "$(elastic-package stack shellinit)" && elastic-package test pipeline --generate -v`
```
--- Test results for package: tenable_sc - START ---
╭────────────┬───────────────┬───────────┬───────────────────────────────────────────────────┬────────┬──────────────╮
│ PACKAGE    │ DATA STREAM   │ TEST TYPE │ TEST NAME                                         │ RESULT │ TIME ELAPSED │
├────────────┼───────────────┼───────────┼───────────────────────────────────────────────────┼────────┼──────────────┤
│ tenable_sc │ asset         │ pipeline  │ (ingest pipeline warnings test-asset.log)         │ PASS   │ 322.975375ms │
│ tenable_sc │ asset         │ pipeline  │ test-asset.log                                    │ PASS   │  65.882958ms │
│ tenable_sc │ plugin        │ pipeline  │ (ingest pipeline warnings test-plugin.log)        │ PASS   │   302.7675ms │
│ tenable_sc │ plugin        │ pipeline  │ test-plugin.log                                   │ PASS   │  58.123791ms │
│ tenable_sc │ vulnerability │ pipeline  │ (ingest pipeline warnings test-vulnerability.log) │ PASS   │ 323.123834ms │
│ tenable_sc │ vulnerability │ pipeline  │ test-vulnerability.log                            │ PASS   │  120.73975ms │
╰────────────┴───────────────┴───────────┴───────────────────────────────────────────────────┴────────┴──────────────╯
--- Test results for package: tenable_sc - END   ---
Done
```
3. Run system tests for package. Should pass successfully.
`eval "$(elastic-package stack shellinit)" && elastic-package test system --generate -v`
```
--- Test results for package: tenable_sc - START ---
╭────────────┬───────────────┬───────────┬───────────┬────────┬───────────────╮
│ PACKAGE    │ DATA STREAM   │ TEST TYPE │ TEST NAME │ RESULT │  TIME ELAPSED │
├────────────┼───────────────┼───────────┼───────────┼────────┼───────────────┤
│ tenable_sc │ asset         │ system    │ default   │ PASS   │   36.7143465s │
│ tenable_sc │ plugin        │ system    │ default   │ PASS   │ 39.125610417s │
│ tenable_sc │ vulnerability │ system    │ default   │ PASS   │  34.20195875s │
╰────────────┴───────────────┴───────────┴───────────┴────────┴───────────────╯
--- Test results for package: tenable_sc - END   ---
Done
```



## Related issues


- Closes https://github.com/elastic/integrations/issues/10419
